src/assets/scripts/client/airport/AirportModel.js

```diff
@@ -463,6 +464,27 @@ export default class AirportModel {
         });
     }
 
+    /**
+     * @for AirportModel
+     * @method prepareCircularRestrictedArea
+     * @param center {array<number>}
+     * @param radius {number}   circle radius, in nm
+     */
+    getCircularCoordinates(center, radius) {
+        const NUM_POINTS = 32;
+        const delta = tau() / NUM_POINTS;
+        let coords = [];
```

Comment:
const

Reply:
@grandsbor Arrays can have their _contents_ modified through `.push()`, `.pop()`, `.unshift()`, etc, even if they are a const. The const will only ensure that there are no _reassignments_ of the value, such as later doing `coords = 'words';` or `coords = [3.14159].

You should actually be seeing a lint error that points out the preference to use const here. If you install eslint addons for your IDE, it will tell you any issues like these:

![image](https://user-images.githubusercontent.com/5103735/51802917-d0432200-221c-11e9-9bcb-72e2e2abc9fe.png)

Reply:
Thanks!
(Never written in Node before)